server/data/mappers/archiveGoalMapper.test.ts

```diff
@@ -0,0 +1,24 @@
+import type { ArchiveGoalDto } from 'dto'
+import type { ArchiveGoalRequest } from 'educationAndWorkPlanApiClient'
+import ReasonToArchiveGoalValue from '../../enums/ReasonToArchiveGoalValue'
+import toArchiveGoalRequest from './archiveGoalMapper'
+
+describe('archiveGoalMapper', () => {
+  it('should map from DTO to request object', () => {
+    const prisonNumber = 'FOO'
```

Comment:
Minor, but we try to use test data that is representative of the use case
In this case `FOO` would not be a valid prison number. Can we change it to something that fits the recognised prison number pattern please? Something like `A1234BC` ?